src/components/ViewCourse/sections/ViewCourseInstructors.js

```diff
@@ -66,15 +66,12 @@ const ViewCourseInstructors = (props) => {
                         permissions.edit && !instructor.patronGroupObject && (
                           <FormattedMessage id="ui-courses.editInstructor">
                             {ariaLabel => (
-                              <Button
+                              <IconButton
                                 aria-label={ariaLabel}
-                                buttonStyle="primary"
+                                icon="edit"
                                 id={`clickable-edit-instructor-${index}`}
-                                marginBottom0
                                 to={`/cr/instructors/${courseListingObject.id}/${record.id}/${instructor.id}/edit`}
-                              >
-                                <FormattedMessage id="ui-courses.button.editInstructor" />
```

Comment:
Can this and other translation keys be removed completely?

Reply:
Probably. I actually thought we were using the same translation key for the aria-label but I now see the difference in `id`s. I'll Ctrl+F through the repo to double-check we're not using them anywhere and delete them if we're not.

Reply:
Fixed in new commit.

Reply:
Yep, looks good.